src/background/services/background.ts

```diff
@@ -131,7 +130,10 @@ export class Background {
               )
 
             case ContentToBackgroundAction.IS_TAB_MONETIZED:
-              this.tabEvents.onUpdatedTab(message.payload)
+              await this.tabEvents.onUpdatedTab({
+                ...message.payload,
+                tabId: sender.tab!.id!
+              })
```

Comment:
Instead of passing the `tabId` directly, let's just pass in the sender and use the `getTabId` utility, just like we do in the monetization service:

https://github.com/interledger/web-monetization-extension/blob/074112917a157fa956796bfbfc2c7b59909ae621/src/background/services/monetization.ts#L114


```suggestion
              await this.tabEvents.onUpdatedTab(message.payload, sender)
```